plugins/Cargo.toml

```diff
@@ -9,12 +9,12 @@ repository = "https://github.com/centricular/webrtcsink/"
 build = "build.rs"
 
 [dependencies]
-gst = { package = "gstreamer", version = "0.18", features = ["v1_20", "ser_de"] }
-gst-app = { package = "gstreamer-app", version = "0.18", features = ["v1_20"] }
-gst-video = { package = "gstreamer-video", version = "0.18", features = ["v1_20", "ser_de"] }
-gst-webrtc = { package = "gstreamer-webrtc", version = "0.18", features = ["v1_20"] }
-gst-sdp = { package = "gstreamer-sdp", version = "0.18", features = ["v1_20"] }
-gst-rtp = { package = "gstreamer-rtp", version = "0.18", features = ["v1_20"] }
+gst = { git="https://gitlab.freedesktop.org/gstreamer/gstreamer-rs", rev="61a3f5296c3093e9811d883256ad38bd5483b6db", package = "gstreamer", features = ["v1_22", "ser_de"] }
```

Comment:
I do not think we should always depend on 1.22 but instead have it as an option of this crate.